src/book/03-guides/03-hardware/04-DARwIn-OP2-guide.mdx

```diff
@@ -0,0 +1,328 @@
+
+Now that we have a recovery USB created with all the right files placed upon it, it is now time to run it on the PICO-CV01 board.
+
+1. Boot up your PICO-CV01 in the BIOS menu (this is usually done by pressing either f12 or delete), locate the **boot** tab and search for **boot sequence**.
+   Select the Clonezilla USB as the first selection for the sequence, save the changes and restart.
+   Once this restarts, Clonezilla should boot up straight away.
+
+2. When Clonezilla boots, a menu screen will appear. Execute the following instructions:
+
+   1. Clonezilla
+
+      - Select the default option **'Clonezilla live (default settings. VGA 800x600)'**
+
+   2. Language
+
+      - Choose language **'Whichever is your preferred language'**
+
+   3. Configuring Console-data
+
+      - Select default option **'Don't touch keymap'**
+
+   4. Start Clonezilla
+
+      - **'Start_Clonezilla Start Clonezilla'**
+
+   5. Clonezilla
+
+      - Select default option **'device-image work with disks or partitions using images'**
+
+   6. Mount Clonezilla Image Directory
+
+      - Select the following **'skip Use existing/home/partimag (Writable device. E.g. hard disk/USB drive)'**
+
+   7. Clonezilla - OpenSource Clone System (OCS)
+
+      - Select the following **'Beginner Beginner mode'** then **'Accept the default options'**
+
+   8. Clonezilla Select mode
+
+      - Select the following **'restoredisk Restore_an_image_to_local_disk'**
+
+   9. Clonezilla - OpenSource Clone System (OCS)
+
+      - Choose the image file to restore **'this will be your recovery image'**
+
+   10. Clonezilla - OpenSource Clone System (OCS)
+
+       - Choose the target disk **'this will be your SSD (make sure to check your SSD capacity to see if the recovery files will fit!)'**
+
+   11. Starting to Restore Image to Device
+
+       - Wait until this is complete.
+
+   12. Completed Restoration
+
+       - Press enter to continue
+
+   13. Power Off or Reboot
+
+       - Select one of the following options **'(0) Power Off, (1) Reboot'**
+
+   14. When the DARwIn-OP2 has successfully shut down, remove the USB and reboot the robot.
+
+For more information about this process check out the [Robotis Clonezilla recovery page](https://emanual.robotis.com/docs/en/platform/op/maintenance/).
+
+### Calibrating the DARwIn-OP2 (Ball Calibration)
+
+If the DARwIn-OP2 is unable to track the ball well, then vision needs calibrating.
+
+1. Using a portable monitor, plug in a HDMI to mini HDMI cable (mini is to be connect to the DARwIn-OP2, normal HDMI being connected to the monitor).
+
+   <Alert>
+      Do not power the monitor from the DARwIn-OP2.
+   </Alert>
+
+2. Start the DARwIn-OP2 so the standard demo program is running. Once booted, it should be in the initial sitting pose.
+
+3. Connect a keyboard and mouse to the DARwIn-OP2 and use them to connect to the robot via ssh in the terminal: `ssh robotis@192.168.123.1`, **with password `111111` (six ones)**.
```

Comment:
This looks like it needs updating like the walk tuner section. I think it could be useful to mention somewhere both methods (using the monitor and keyboard to directly access the darwin computer, and using ssh on a separate computer to connect to the darwin computer).

Reply:
got rid of ssh